src/Compilers/CSharp/Test/Semantic/FlowAnalysis/FlowDiagnosticTests.cs

```diff
@@ -2550,5 +2550,57 @@ void M()
                 //         r = (y+1)..(y=0);
                 Diagnostic(ErrorCode.ERR_UseDefViolation, "y").WithArguments("y").WithLocation(9, 14));
         }
+
+        [Fact]
+        public void FieldAssignedInLambdaOnly()
```

Comment:
Was there any actual behavioral change here, or this just documenting existing behavior?

Reply:
Just testing existing behavior to avoid regressions.

---
In reply to: [241560302](https://github.com/dotnet/roslyn/pull/31703#discussion_r241560302) [](ancestors = 241560302)